src/GitTfs/Core/GitTfsRemote.cs

```diff
@@ -979,7 +979,7 @@ private void WithWorkspace(TfsChangesetInfo parentChangeset, Action<ITfsWorkspac
             }
             else
             {
-                Tfs.WithWorkspace(WorkingDirectory, this, parentChangeset, action);
+                Tfs.WithWorkspace(WorkingDirectory, this, parentChangeset, action, workspaceOwner);
```

Comment:
This means that feature is not available for the subtrees workspace :-(

Reply:
Yeah, as you mentioned before a would like to limit the scope of this PR maybe in the future I can take this feature to the other commands as well.